docs/tutorial/go.rst

```diff
@@ -0,0 +1,361 @@
+    generated ``.rock`` file.
+
+Pack the rock:
+
+.. literalinclude:: code/go/task.yaml
+    :language: bash
+    :start-after: [docs:pack]
+    :end-before: [docs:pack-end]
+    :dedent: 2
+
+.. note::
+
+    Depending on the network, this step can take a couple of minutes to finish.
+
+    ``ROCKCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS`` is required whilst the ``go-framework``
+    extension is experimental.
+
+Once Rockcraft has finished packing the Go rock, we'll find a new file in
+the working directory (an `OCI <OCI_image_spec_>`_ archive) with the ``.rock``
+extension:
+
+.. literalinclude:: code/go/task.yaml
+    :language: bash
+    :start-after: [docs:ls-rock]
+    :end-before: [docs:ls-rock-end]
+    :dedent: 2
+
+.. note::
+    If we changed the ``name`` or ``version`` in ``rockcraft.yaml`` or are not
+    on an ``amd64`` platform, the name of the ``.rock`` file will be different.
+
+    By default, the ``go-framework`` will use the ``bare`` base.
+    The ``bare`` base is recommended for production applications because
+    the image size will be smaller, and the risk of security issues lower.
+    However, we may want to change the base to ``ubuntu@24.04`` during the
+    development process to benefit from the extra utilities provided.
+
+
+Run the Go rock with Docker
+===========================
+
+
+We already have the rock as an `OCI <OCI_image_spec_>`_ archive. Load the
+image into Docker:
+
+.. literalinclude:: code/go/task.yaml
+    :language: bash
+    :start-after: [docs:skopeo-copy]
+    :end-before: [docs:skopeo-copy-end]
+    :dedent: 2
+
+
+Check that the image was successfully loaded into Docker:
+
+.. literalinclude:: code/go/task.yaml
+    :language: bash
+    :start-after: [docs:docker-images]
+    :end-before: [docs:docker-images-end]
+    :dedent: 2
+
+The output should list the Go container image, along with its tag, ID and
+size:
+
+..  code-block:: text
+    :class: log-snippets
+
+    REPOSITORY       TAG       IMAGE ID       CREATED         SIZE
+    go-hello-world   0.1       f3abf7ebc169   5 minutes ago   15.7MB
+
+.. note::
+    The size of the image reported by Docker is the uncompressed size which is
+    larger than the size of the compressed ``.rock`` file.
+
+Now we're finally ready to run the rock and test the containerised Go
+application:
+
+.. literalinclude:: code/go/task.yaml
+    :language: text
+    :start-after: [docs:docker-run]
+    :end-before: [docs:docker-run-end]
+    :dedent: 2
+
+Use the same ``curl`` command as before to send a request to the Go
+application's root endpoint which is running inside the container:
+
+.. literalinclude:: code/go/task.yaml
+    :language: text
+    :start-after: [docs:curl-go-rock]
+    :end-before: [docs:curl-go-rock-end]
+    :dedent: 2
+
+The Go application should again respond with ``Hello, world!``.
+
+
+View the application logs
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When deploying the Go rock, we can always get the application logs via
```

Comment:
```suggestion
When deploying the Go rock, we can always get the application logs with
```

We must [avoid Latin](https://docs.ubuntu.com/styleguide/en/#latin-words-and-phrases).